### PR TITLE
Add ForexRatesTable prop to filter single currency

### PR DESCRIPTION
**Pull request by KamuelaFranco**

### CODE REVIEW

**Comment by msk-** [comment]
What's the best way to test this?

File: src/components/ForexRatesTable.js [hunk 1]
```diff
@@ -7,6 +7,17 @@ import { uid } from 'react-uid';
...
+export const onlyCurrency = (forexRates, currencyPair) => {
+  if (!currencyPair) {
+    return forexRates;
+  }
+  return forexRates.filter((rate) => rate.currencyPair === currencyPair);
+};
+
+export const setForexRatesOnly = (
+  setForexRates, currencyPair, onlyCurrencyFunc = onlyCurrency,
+) => (newRates) => setForexRates(onlyCurrencyFunc(newRates, currencyPair));
```
Comment (msk-) [inline src/components/ForexRatesTable.js]: This doesn't really look like it's used anywhere except the test? Do we need it?

Comment (KamuelaFranco) [reply src/components/ForexRatesTable.js]: I can get rid of it, it was part of an earlier attempt.

**Comment by KamuelaFranco** [comment]
> What's the best way to test this?

@msk- Clone locally, `npm start`, and then `npm run test-e2e:local`.

**Pull request merged**